tests/test_processors.py

```diff
@@ -1171,3 +1172,90 @@ def test_replace_by_key_is_optional(self):
         assert {"msg": "hi", "foo": "bar"} == EventRenamer("msg", "missing")(
             None, None, {"event": "hi", "foo": "bar"}
         )
+
+
+class TestCallsiteNamespaceAddr:
+    def test_simple_lookup(self):
+        """
+        Simple verification of path interogation
+        """
+        assert "{}.{}.{}".format(
+            self.__module__,
+            self.__class__.__qualname__,
+            sys._getframe().f_code.co_name,
+        ) == CallsiteNamespaceAddr().get_qual_name(sys._getframe())
+
+    async def test_async_lookup(self):
+        """
+        Simple verification of path interogation against async function
+        """
+        assert "{}.{}.{}".format(
+            self.__module__,
+            self.__class__.__qualname__,
+            sys._getframe().f_code.co_name,
+        ) == CallsiteNamespaceAddr().get_qual_name(sys._getframe())
+
+    def test_processor(self):
+        """
+        Ensure `CallsiteNamespaceAddr` Processor can be enabled, and
+        that the ``namespace`` details are present and correct in a
+        log entry
+        """
+        cf = structlog.testing.CapturingLoggerFactory()
+        structlog.configure(
+            logger_factory=cf,
+            processors=[
+                structlog.processors.CallsiteNamespaceAddr(),
+                structlog.processors.JSONRenderer(),
+            ],
+        )
+        structlog.get_logger().info("test!")
+        assert (
+            "{}.{}.{}".format(
+                self.__module__,
+                self.__class__.__qualname__,
+                sys._getframe().f_code.co_name,
+            )
+            == json.loads(cf.logger.calls.pop().args[0])["namespace"]
+        )
+
+    def test_level_limitor(self):
```

Comment:
not a native speaker but I don't think that's a word :)

```suggestion
    def test_level_limiter(self):
```